src/db_iterator.rs

```diff
@@ -529,6 +529,9 @@ impl Iterator for DBWALIterator {
     type Item = (u64, WriteBatch);
 
     fn next(&mut self) -> Option<(u64, WriteBatch)> {
+        if !self.valid() {
```

Comment:
Because of the GetUpdatesSince() fix from facebook/rocksdb#9459,
the resulting iterator from `DB::get_updates_since()` will have its status Invalid
on empty result.